petab/visualize/helper_functions.py

```diff
@@ -1,10 +1,10 @@
 """
 helper_functions.py
-=========
+===================
```

Comment:
"helper_functions.py" is not an overwhelming title. "Helper functions"?

Reply:
I didn't do it ;). Totally agreed, but to be changed elsewhere. Many docstrings in petab.visualization still need some makeover. I think there are still quite some things in motion, so I only fixed more severe sphinx issues.